### DISCUSSION TITLE
是否只支持树莓派4？

### DISCUSSION THREAD

**Discussion by Wenaka2004** [SinKy-Yan/zhenxunbot-docker#21] [Q&A] [answered]
用树莓派3，32位系统运行出现如下报错
![WVCHT@ X ~SQIP_S}NS$O$Y](https://user-images.githubusercontent.com/121840779/210294619-e03ae96c-c15b-4cdd-9675-d254453ea182.png)
看样子是不支持armv7l的处理器？

**Comment by SinKy-Yan** [comment, marked as answer]
不是只支持4，是支持arm64内核的，你的3上的CPU已经支持了，应该是你的系统是官方的32位系统吧，你可以试试装个64位的
这个问题我有时间也会试试构建一个适配arm v7的镜像

**Reply by Wenaka2004** [reply]
好的我去试试
感谢
